Running-Mastodon/PgBouncer-guide.md

```diff
@@ -87,7 +78,7 @@ listen_addr = 127.0.0.1
 listen_port = 6432
 ```
 
-Put `md5` as the `auth_type` (assuming you're using the md5 format in `userlist.txt`):
+Put `md5` as the `auth_type`:
```

Comment:
Not sure if it is worth mentioning because it didn't work for me. At least not with the formula you mentioned ('md5'+md5(`user`+`password`)).
But as I mentioned before, this was my experience and maybe others will find different results. The only reason I am a *bit* more sure is because this was a fresh install of Debian.